sw/device/silicon_creator/rom_ext/keys/fake/rom_ext_dev_key_0_rsa_3072_exp_f4.h

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXT_KEYS_FAKE_ROM_EXT_DEV_KEY_0_RSA_3072_EXP_F4_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_ROM_EXT_KEYS_FAKE_ROM_EXT_DEV_KEY_0_RSA_3072_EXP_F4_H_
```

Comment:
Out of curiosity, how did you generate this header, was it manually?
@pamaury will have to generate keys for a the rom key set.

Reply:
Hi @engdoreis!, I used the `opentitantool` RSA `generate` and `show` commands. For example:

```console
bazel-bin/sw/host/opentitantool/opentitantool rsa key show \
    sw/device/silicon_creator/rom_ext/keys/fake/rom_ext_prod_key_0_rsa_3072_exp_f4.pub.der 
```

Unfortunately, the tool does not yet support printing keys in C header format. It would be great if this could be added in a follow up change.

Reply:
Thanks Miguel, @pamaury is plaining to add this feature in opentitantool.

Reply:
The code in !17346 can now generate the header files from keys.

Reply:
Awesome! Can you move that specific change into a separate PR? That way it can be reviewed separately.

Reply:
I have moved it to a separate PR #17398 and also slightly tweaked the output.

Reply:
Thanks @pamaury!